### PR TITLE
Various Bugfixes

### LINKED ISSUE #349: Check if grid-gutter-width can be used for responsive size padding
Would it be appropriate to use `@grid_gutter-width` in place of `30px` [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/static/css/misc.less#L789)?

### LINKED ISSUE #517: Change `respond-to-retina` to `respond-to-dpi`
See https://github.com/cfpb/cfgov-refresh/pull/508#discussion_r29686797

### LINKED ISSUE #519: Normalize quotes in jinja template
See https://github.com/cfpb/cfgov-refresh/pull/508#discussion_r29687751

### LINKED ISSUE #521: Stack long class lists
See https://github.com/cfpb/cfgov-refresh/pull/508#discussion_r29686942

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Most of these changes don't actually change much, but I wanted to close some of our Github issues in between tasks.

- Closes #521
- Closes #519 
- Closes #517 
- Closes #349

### CODE REVIEW

**Comment by anselmbradford** [comment]
:+1:

File: static/css/cf-enhancements.less [hunk 1]
```diff
@@ -146,7 +148,7 @@
     - cfgov-cf-enhancements
 */
 
-.respond-to-retina(@rules) {
+.respond-to-dpi(@rules) {
     @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```
Comment (jimmynotjim) [inline static/css/cf-enhancements.less]: My suggestion on the PR is to pass the pixel ratio as a param (2 or 3) and change the media query based on that so we can support @3x devices with this mixin.

Comment (KimberlyMunoz) [inline static/css/cf-enhancements.less]: I see what you're saying. I thought the issue was the retina was too Apple specific.

Comment (KimberlyMunoz) [inline static/css/cf-enhancements.less]: Fixed. Also, TIL that some Samsung devices have 3x standard dpi displays.

Comment (KimberlyMunoz) [inline static/css/cf-enhancements.less]: ⛄️

**Comment by jimmynotjim** [comment]
:+1: feel free to squash/merge when you're ready

**Pull request merged**